package.json

```diff
@@ -63,6 +63,7 @@
     "react-native-modal": "^11.5.4",
     "react-native-paper": "^3.5.1",
     "react-native-reanimated": "~1.4.0",
+    "react-native-render-html": "^4.2.0",
```

Comment:
any new packages used, make sure you mention them in the doc under section "External libraries"

Reply:
🥇

Reply:
Nice catch!